package.json

```diff
@@ -57,10 +57,10 @@
     "open": "concurrently \"http-server -a localhost -p 3000\" \"open http://localhost:3000\"",
     "backend": "python3 manage.py runserver",
     "bundle-plotly": "cd node_modules/plotly.js && npm i && npm run custom-bundle -- --traces scatter,scatter3d,surface,table,histogram,image,bar,indicator",
-    "rq-watch": "OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES python3 PYTHON/watch_worker.py",
-    "rq-flojoy": "cd PYTHON && OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES python3 worker.py",
+    "rq-watch": "OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES python3 PYTHON/rq_worker.py flojoy-watch",
+    "rq-flojoy": "OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES python3 PYTHON/rq_worker.py flojoy",
     "start-project": "concurrently --prefix \"[{time}-{name}]\" -n \"React,Django,RQ-watch,RQ-flojoy\" -c auto  \"npm:start\" \"npm:backend\" \"npm:rq-watch\" \"npm:rq-flojoy\"",
-    "start-project:win": "concurrently --prefix \"[{time}-{name}]\" -n \"React,Django,RQ-watch,RQ-flojoy\" -c auto  \"npm:start\" \"npm:backend\" \"rqworker.exe -w rq_win.WindowsWorker flojoy-watch\" \"cd PYTHON && rqworker.exe -w rq_win.WindowsWorker flojoy\"",
+    "start-project:win": "concurrently --prefix \"[{time}-{name}]\" -n \"React,Django,RQ-watch,RQ-flojoy\" -c auto  \"npm:start\" \"python manage.py runserver\" \"python PYTHON/rq_worker.py flojoy-watch\" \"python PYTHON/rq_worker.py flojoy\"",
```

Comment:
Can you please use python3 to start the workers.

Reply:
@yassineMoumen  Thanks for the comment. However as far as I investigated, in Windows `python3` refers to only the default installed python.exe file and it's not possible to use `python3` with venv as venv is created only with `python.exe` script not `python3.exe` script.